tests/core/test_test.py

```diff
@@ -0,0 +1,213 @@
+import asyncio
+import json
+import pathlib
+import pytest
+import logging
+from typing import Text, Any, Dict
+
+from rasa.core.agent import Agent
+from rasa.core.policies.ensemble import SimplePolicyEnsemble
+from rasa.core.policies.rule_policy import RulePolicy
+import rasa.core.test
+from rasa.shared.core.constants import ACTION_LISTEN_NAME
+from rasa.shared.core.domain import Domain
+from rasa.shared.core.events import ActionExecuted, UserUttered
+from rasa.shared.core.generator import TrackerWithCachedStates
+from rasa.shared.nlu.interpreter import RegexInterpreter
+
+
+@pytest.fixture(scope="function")
+def out_directory(tmpdir: pathlib.Path):
```

Comment:
I guess not. I just thought it looked cleaner to have the args to test_test be more informative, e.g., out_directory instead of tmpdir, but I can change it.

Reply:
make sense - and I'm generally all for more informative names. `tmpdir` is pretty widely used around the code base for this purpose though.